gui/kitchen-info.lua

```diff
@@ -0,0 +1,414 @@
+-- Adds more info to the Kitchen screen.
+--@ enable = true
+--[====[
+
+gui/kitchen-info
+===================
+Adds more info to the Kitchen screen.
+
+Usage::
+
+    gui/kitchen-info enable|disable|help
+    enable|disable gui/kitchen-info
+
+]====]
+
+gui = require 'gui'
+dialogs = require 'gui.dialogs'
+gps = df.global.gps
+world = df.global.world
+
+local kitchen_overlay = defclass(kitchen_overlay, gui.Screen)
+kitchen_overlay.focus_path = 'kitchen_overlay'
```

Comment:
```suggestion
kitchen_overlay.ATTRS {
    focus_path='kitchen_overlay',
}
```

Reply:
But it appears like this (before change) in many other gui scripts?

Reply:
There's a lot of legacies in our current codebase that are not necessarily worth perpetuating.

Reply:
@myk002 also said: "No functional difference as written, but ATTRS is the official API for setting properties on your class."

Ok, changing it.